application/app/views/projects/show/_download_files.html.erb

```diff
@@ -29,7 +29,19 @@
 
         <div class="col-md-2 file-status d-flex justify-content-end gap-1">
           <div class="d-flex flex-column justify-content-center align-items-center">
-            <%= status_badge(file.status, title: file.end_date) %>
+            <%= button_tag type: 'button',
```

Comment:
update the CSS so that the pointer looks like you can click. In all cases where you can open the modal